packages/ffe-details-list-react/package.json

```diff
@@ -36,7 +35,8 @@
     "react-dom": "^16.2.0"
   },
   "peerDependencies": {
-    "@sb1/ffe-details-list": "^9.0.2",
+    "@sb1/ffe-grid": "^10.0.1",
+    "@sb1/ffe-grid-react": "^9.0.1",
```

Comment:
The CI build naturally doesn't install peer-deps so I'll try adding the to dev-deps as well in order to fix the build